### ISSUE TITLE
Error: unable to generate the customized disk image

### ISSUE THREAD

**Issue by markellerby** [PJ-Singh-001/Cubic#170]
I am trying to build a custom Ubuntu 22.04 with some preseed options.
When it comes to building the image, it gets to "Generate the customized disk image" it gets to 1.8% and errs with "Error: unable to generate the customized disk image"
Is there an error log file somewhere, so I can find out where it is failing?
thanks
![Screenshot from 2023-03-08 13-07-53](https://user-images.githubusercontent.com/127313975/223721213-b1701f40-aac1-4519-ad31-f776a03550c7.png)

**Comment by PJ-Singh-001** [comment]
You can run Cubic from the command line using

    cubic --verbose

Usually, the issue is that you do not have enough disk space. You should check available disk space using

    df -h <path to cubic project directory>

You might want to check this repeatedly while on the Generate page.

**Comment by markellerby** [comment]
Thanks, I got the log using "cubic -v". The fatal error is:
```
Permission denied\nlibburn : FATAL :
                                            Burn run failed\nxorriso : FATAL :  
                                            -abort_on 'FAILURE' encountered     
                                            'FATAL' during image                
                                            writing\nxorriso : NOTE : libburn   
                                            has now been urged to cancel its    
                                            operation\nxorriso
```
I'm running it as a non-root user and the $HOME/cubic directory is writeable. The custom-disk dir is owned by me, but custom-root is owned by root:root. I haven't changed anything though so I assume this is normal?
I have plenty of disk space: 294GB
If I create a fresh VM, and install cubic on that, it all works OK. So I can do that. But have you any suggestions on the above error, out of curiosity?
thanks

**Comment by PJ-Singh-001** [comment]
Ownership should be as shown below (`user_1` is your user id).
Are your ownerships the same?
(Yes, `custom-root` should be owned by root).

    tree -L 1 -p -u cubic_project_directory

    [drwxrwxr-x user_1  ]  cubic_project_directory
    ├── [-rw-rw-r-- user_1  ]  cubic.conf
    ├── [drwxrwxr-x user_1  ]  custom-disk
    ├── [drwxr-xr-x root    ]  custom-root
    ├── [-rw-rw-r-- user_1  ]  ubuntu-22.04-desktop-amd64-2023.03.14.iso
    ├── [-rw-rw-r-- user_1  ]  ubuntu-22.04-desktop-amd64-2023.03.14.md5
    ├── [-rw-rw-r-- user_1  ]  partition-1.img
    └── [-rw-rw-r-- user_1  ]  partition-2.img

Do you also have this problem if you create a new Cubic project _inside_ your home directory, for example `\home\user_1\Test_Project`, where `user_1` is your home directory and `Test_Project` is a new directory owed by you?

**Comment by markellerby** [comment]
I was running it in $HOME/cubic, but I tried creating a new directory $HOME/cubic2, and using the same ISO and preseed and boot files, it created the image without any problems!
I ran a diff on the cubic.conf in both directories and it is the same apart from the PATH and the iso checksums etc. Also all the file ownerships and modes are the same.
So it's a mystery, but that's OK, I'll just use the cubic2 directory :)
thanks for your help.
